### ISSUE TITLE
Packaged version of `cthyb` fails -- illegal hardware instruction

### ISSUE THREAD

**Issue by DerWeh** [TRIQS/cthyb#90]
I installed `triqs` + `cthyb` (cthyb Version: 1.5) via package manager

```bash
sudo apt-get update && sudo apt-get install -y software-properties-common apt-transport-https curl
source /etc/lsb-release
curl -L https://users.flatironinstitute.org/~ccq/triqs/master/$DISTRIB_CODENAME/public.gpg | sudo apt-key add -
sudo add-apt-repository "deb https://users.flatironinstitute.org/~ccq/triqs/unstable/$DISTRIB_CODENAME/ /"
sudo apt-get update && sudo apt-get install -y triqs cthyb
```

However, initializing the `cthyb` solver fails:
```python
# fail.py
from triqs_cthyb import Solver as Ct_hyb
ct_hyb = Ct_hyb(beta=100., gf_struct=[('up', [0]), ('dn', [0])])
```
yields

```bash
❯ python fail.py
Starting on 1 Nodes at : 2018-06-06 00:00:45.293758
[1]    97 illegal hardware instruction (core dumped)  python fail.py
```

Please find my [cpuinfo](https://github.com/TRIQS/cthyb/files/2074168/cpuinfo.txt) attached.

**Comment by HugoStrand** [comment]
Dear Nils,

Are the debian packages built with -march=native? We should only require the basic x86 instruction set for them to be binary compatible with older systems.

Best, Hugo

**Comment by HugoStrand** [comment]
Dear @Wentzell,

Any update on this?

Best, Hugo

**Comment by Wentzell** [comment]
There was indeed an `-march=native` flag added by cmake that should not have been there.

This should be fixed by 073045191cb99d4fb51d280b213a26d39ec385b0. Reopen if it still persists.
